java/core/src/java/org/apache/orc/DigestConf.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.orc;
+
+import com.tdunning.math.stats.TDigest;
+
+public class DigestConf {
+
+  public static final DigestConf NO_CREATE_DIGEST = new DigestConf(false, 0);
+
+  public static final DigestConf DEFAULT_DIGEST = new DigestConf(true, 1000);
+
+  private final boolean createDigest;
+
+  private final double digestCompression;
```

Comment:
What does it mean? Does t-digest supports compression natively (using standard compression algorithm) or simple serialization?